src/parquet/file/metadata.h

```diff
@@ -91,6 +91,32 @@ class PARQUET_EXPORT RowGroupMetaData {
 
 class FileMetaDataBuilder;
 
+struct PARQUET_EXPORT FileVersion {
```

Comment:
It's small, and semantically part of `FileMetaData`. We could also make it a nested class `FileMetaData::Version`